tslint_base.json

```diff
@@ -16,6 +16,10 @@
         "singleQuote": true,
         "trailingComma": "all"
       }
+    ],
+    "import-blacklist": [
+      true,
+      [".*\\/assets\\/icons$"]
```

Comment:
I guess we should also add `".*\\/assets\\/icons/social$"`.

Do we also need a rule for implicitly using `index` or `index.tsx` in the path?

Reply:
> I guess we should also add `".*\\/assets\\/icons/social$"`.
>
> Do we also need a rule for implicitly using `index` or `index.tsx` in the path?

Yes, I will add it as well.